source/agora/node/FullNode.d

```diff
@@ -558,9 +561,22 @@ public class FullNode : API
         {
             log.error("Error sending updated block headers:{}", e);
         }
-        this.network.getBlocksFrom(
-            nextHeight,
-            &this.addBlocks);
+
+        const Height expected = this.ledger.expectedHeight(this.clock.utcTime());
+        if (expected < this.ledger.height)
+            this.log.warn("Our current Ledger state is ahead of the expected height (current: {}, expected: {}",
+                          this.ledger.height, expected);
+        else if (expected > this.ledger.height)
+        {
+            this.log.info("Ledger has fell behind, missing {} blocks (current height: {})",
```

Comment:
```suggestion
            this.log.info("Ledger has fallen behind, missing {} blocks (current height: {})",
```

Reply:
[I should have known](https://www.youtube.com/watch?v=9BXNKyoW_Ow).